tools/test/test262/excludelist.orig.xml

```diff
@@ -3875,31 +3868,7 @@
     <test id="language/expressions/assignment/dstr/array-rest-iter-thrw-close-err"><reason>TODO</reason></test>
     <test id="language/expressions/assignment/dstr/array-rest-lref-err"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-id-init-fn-name-arrow"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-id-init-fn-name-class"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-id-init-fn-name-cover"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-id-init-fn-name-fn"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-id-init-fn-name-gen"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-prop-elem-target-obj-literal-prop-ref-init"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-prop-elem-target-obj-literal-prop-ref-init-active"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-computed-property"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-computed-property-no-strict"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-descriptors"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-empty-obj"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-getter"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-getter-abrupt-get-error"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-number"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-order"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-put-const"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-same-name"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-skip-non-enumerable"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-str-val"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-symbol-val"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-to-property"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-to-property-with-setter"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-val-null"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-val-undefined"><reason>TODO</reason></test>
-    <test id="language/expressions/assignment/dstr/obj-rest-valid-object"><reason>TODO</reason></test>
+    <test id="language/expressions/assignment/fn-name-lhs-cover"><reason>TODO</reason></test>
```

Comment:
I believe this test case would not be add.
Could you check https://github.com/Samsung/escargot/pull/550/files#diff-df19a918b821d0641fd253e624e68699R2859 here?

Reply:
I fixed it. Thanks!